api/src/main/java/jakarta/json/bind/annotation/JsonbNumberFormat.java

```diff
@@ -36,6 +37,11 @@
  *   <li> package </li>
  * </ul>
  *
+ * <p>Note that even when specifying a pattern format your JSON is depending on the Locale of the JVM you run on.
```

Comment:
This sentence is difficult to understand without punctuation to identify which of the different parts go together and which don't. Here is one suggestion to clarify it while leaving it mostly as written,

```suggestion
 * <p>Note that even when specifying a pattern, the format of the JSON value depends on the Locale of the JVM by default.
```

Reply:
txs for the review and suggestion!